### PR TITLE
Elasticsearch 5.0.0-beta1

### PR DESCRIPTION
**Pull request by jasontedor**
This pull request upgrades the devel release for Elasticsearch from version 5.0.0-alpha5 to 5.0.0-beta1, the [latest pre-release version of the Elasticsearch 5.x series](https://www.elastic.co/blog/elasticsearch-5-0-0-beta1-released) as of 2016-09-21.

### CODE REVIEW

File: Formula/elasticsearch.rb [hunk 1]
```diff
@@ -5,20 +5,19 @@ class Elasticsearch < Formula
...
-    url "https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/5.0.0-alpha5/elasticsearch-5.0.0-alpha5.tar.gz"
-    sha256 "4ec8fa57310affa9e0510aa18fb98c8b08f7e7d6f591d29c3e1fffe88d411813"
-    version "5.0.0-alpha5"
+    url "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.0.0-beta1.tar.gz"
+    sha256 "4ff6680b2d053c74835db77dcb03e02340555cd70cae8bb73d3b1f94ddf0147d"
   end
 
   head do
     url "https://github.com/elasticsearch/elasticsearch.git"
-    depends_on java: "1.8"
```
Comment (zmwangx) [inline Formula/elasticsearch.rb]: Revert all these syntax changes.

Comment (zmwangx) [inline Formula/elasticsearch.rb]: Revert all these syntax changes.

File: Formula/elasticsearch.rb [hunk 2]
```diff
@@ -5,20 +5,19 @@ class Elasticsearch < Formula
   sha256 "3ae01140ae7bcbb91436feef381fbed774e36ef6d1e8e6a3153640db82acf4c9"
 
   devel do
-    url "https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/5.0.0-alpha5/elasticsearch-5.0.0-alpha5.tar.gz"
-    sha256 "4ec8fa57310affa9e0510aa18fb98c8b08f7e7d6f591d29c3e1fffe88d411813"
-    version "5.0.0-alpha5"
+    url "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.0.0-beta1.tar.gz"
+    sha256 "4ff6680b2d053c74835db77dcb03e02340555cd70cae8bb73d3b1f94ddf0147d"
```
Comment (zmwangx) [inline Formula/elasticsearch.rb]: Add `version "5.0.0-beta1"`.

**Review by zmwangx — CHANGES_REQUESTED** [review]

File: Formula/elasticsearch.rb (hunk 2 above)
Comment (jasontedor) [inline Formula/elasticsearch.rb]: Why? That fails the audit since it can be extracted from the `url`.

File: Formula/elasticsearch.rb (hunk 1 above)
Comment (jasontedor) [inline Formula/elasticsearch.rb]: I pushed be29182e8cc96fcdbe43cadee50241972829d973.

File: Formula/elasticsearch.rb (hunk 2 above)
Comment (zmwangx) [inline Formula/elasticsearch.rb]: Okay, not sure why `5.0.0-alpha5` can't be extracted but beta can.

**Review by zmwangx — APPROVED** [review]

**Comment by zmwangx** [comment]
Merged as 591ee94. Thank you for your contribution to Homebrew! For future reference, our preferred format of commit subject is `foo 1.2.3-beta1 (devel)`.

**Comment by jasontedor** [comment]
Thank you @zmwangx! :smile:

**Pull request merged**